src/org/opendatakit/briefcase/model/AsJson.java

```diff
@@ -0,0 +1,14 @@
+package org.opendatakit.briefcase.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.Optional;
+
+public interface AsJson {
```

Comment:
Do we still follow the “-able” convention for naming interfaces? Jsonable, or Jsonifyable, or JsonConvertable.